include/common/types.h

```diff
@@ -275,6 +453,53 @@ inline constexpr ValVariant ValueFromType(ValType Type) noexcept {
   }
 }
 
+// >>>>>>>> Const expression to generate value from Interface type
```

Comment:
The comment includes the other types, please make sure the comment is matched to the code section.